src/Thentos/Util.hs

```diff
@@ -77,3 +83,13 @@ readsPrecEnumBoundedShow _ s = f [minBound..]
         (s0, s1) -> if s0 == s' then [(x, s1)] else f xs
       where
         s' = show x
+
+
```

Comment:
This seems to be only used in tests. Maybe it should be moved to the tests Util module?